eureka_ml_insights/models/models.py

```diff
@@ -91,15 +90,15 @@ class EndpointModel(Model):
     num_retries: int = 3
 
     @abstractmethod
-    def create_request(self, text_prompt, query_images=None, system_message=None):
+    def create_request(self, text_prompt, **kwargs):
```

Comment:
Throughout this file create_request has different signatures across the classes, sometimes using args, kwargs, sometimes previous messages and system_messages explicitly.  Is this intentional?

Reply:
Thanks for the feedback. I made them more consistent, basically removing all **kwargs from non-base classes and explicitly defining arguments instead.